Test/QuickCheck/Test.hs

```diff
@@ -361,20 +363,41 @@ success st =
               , not (null s')
               ]
 
-  covers = [ ("only " ++ show (labelPercentage l st) ++ "% " ++ l ++ ", not " ++ show reqP ++ "%")
+  covers = [ ("only " ++ show (labelPrecise (labelCount l st) (numSuccessTests st)) ++ "% " ++ l ++ ", not " ++ show reqP ++ "%")
            | (l, reqP) <- Map.toList (S.labels st)
            , labelPercentage l st < reqP
            ]
 
-  showP p = (if p < 10 then " " else "") ++ show p ++ "% "
```

Comment:
I am not sure what was the purpose of this, but I left it as close to the original functionality as I could.